File: app/src/main/java/com/kevin/kggank/ui/fragment/PicFragment.java
package com.kevin.kggank.ui.fragment;

import com.kevin.kggank.R;
import com.kevin.kggank.base.BaseFragment;

/**
 * Created by kevin on 16/5/13.
 */
public class PicFragment extends BaseFragment {

    public static PicFragment getInstance(){
        return new PicFragment();
    }

    @Override
    protected int getLayout() {
        return R.layout.fragment_pic;
    }

    @Override
    protected void initView() {

    }
}

File: app/src/main/java/com/kevin/kggank/ui/fragment/PicFragment.kt
package com.kevin.kggank.ui.fragment

import com.kevin.kggank.R
import com.kevin.kglib.base.LibBaseFragment

/**
 * Created by kevin on 16/5/13.
 */
class PicFragment : LibBaseFragment() {

    override fun getLayout(): Int {
        return R.layout.fragment_pic
    }

    override fun initView() {

    }

    companion object {

        val instance: PicFragment
            get() = PicFragment()
    }
}
